lib/teamlab/modules/crm/crm_common.rb

```diff
@@ -23,20 +23,28 @@ def get_all_currency_rates
       @request.get(%w[currency rates], {})
     end
 
-    def set_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
+    def create_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
       @request.post(%w[currency rates], fromCurrency: from, toCurrency: to, rate: rate)
     end
 
     def get_currency_rate_by_id(id)
       @request.get(['currency', 'rates', id.to_s], {})
     end
 
+    def get_currency_rate_by_currency(from_currency, to_currency)
+      @request.get(['currency', 'rates', from_currency.to_s, to_currency.to_s])
+    end
+
     def delete_currency_rate_by_id(id)
       @request.delete(['currency', 'rates', id.to_s], {})
     end
 
     def set_is_portal_configured(options = {})
       @request.put(%w[settings], options)
     end
+
+    def change_web_form_key
```

Comment:
Add basic description if possible for new method

Reply:
Deleted method as there is no parameters in [documentation](https://api.onlyoffice.com/portals/method/crm/put/api/2.0/crm/settings/webformkey/change) for changing Website Contact Form key and no information what it is for